connector/connect/server/src/main/scala/org/apache/spark/sql/connect/config/Connect.scala

```diff
@@ -115,4 +115,17 @@ object Connect {
       .version("3.5.0")
       .booleanConf
       .createWithDefault(false)
+
+  val CONNECT_UI_STATEMENT_LIMIT =
+    ConfigBuilder("spark.sql.connect.ui.retainedStatements")
+      .doc("The number of statements kept in the Spark Connect UI history.")
+      .version("3.5.0")
+      .intConf
+      .createWithDefault(200)
```

Comment:
Why the default value is 200? For SQL query retention, the default value is 1000

Reply:
This is what the Thrift Server has for defaults: https://github.com/apache/spark/blob/master/sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala#L1414. Can be changed though. Is 1000 preferred?